Unreal/CarlaUnreal/Plugins/Carla/Source/Carla/Actor/ActorBlueprintFunctionLibrary.cpp

```diff
@@ -1523,14 +1523,14 @@ FColor UActorBlueprintFunctionLibrary::RetrieveActorAttributeToColor(
 // editor crashing while people are testing new actor definitions.
 #if WITH_EDITOR
 #  define CARLA_ABFL_CHECK_ACTOR(ActorPtr)                    \
-  if ((ActorPtr == nullptr) || ActorPtr->IsPendingKill())     \
+  if ((ActorPtr == nullptr) || IsValid(ActorPtr))     \
```

Comment:
```suggestion
  if ((ActorPtr == nullptr) || !IsValid(ActorPtr))     \
```
I think you wanted to look for non valid Actors to throw an error right?

Reply:
You are right